src/Api.Rest.Dtos/Data/AbstractAttributeDefinitionDto.cs

```diff
@@ -12,16 +12,18 @@ namespace Zeiss.PiWeb.Api.Rest.Dtos.Data
 {
 	#region usings
 
-	using Newtonsoft.Json;
+	using System.Text.Json.Serialization;
 	using Zeiss.PiWeb.Api.Rest.Dtos.Converter;
+	using Zeiss.PiWeb.Api.Rest.Dtos.JsonConverters;
 
 	#endregion
 
 	/// <summary>
 	/// Abstract base class of <see cref="CatalogAttributeDefinitionDto"/> ans <see cref="AttributeDefinitionDto"/>.
 	/// It holds attribute's base properties key and description.
 	/// </summary>
-	[JsonConverter( typeof( AttributeDefinitionConverter ) )]
+	[Newtonsoft.Json.JsonConverter( typeof( AttributeDefinitionConverter ) )]

```

Comment:
Why do we need both if the JSON notation is compatible?

Reply:
See https://github.com/ZEISS-PiWeb/PiWeb-Api/pull/157#issuecomment-1210869923